hackjohn.py

```diff
@@ -140,3 +140,27 @@ def get_trailhead_df():
     mmb_response = requests.post(mmb_url, json=payload)
     print('middleman status code', mmb_response.status_code)
     print(mmb_response.text)
+
+## Notifications using IFTTT
+
+# Set enable_ifttt to True to receive IFTTT notification
+enable_ifttt = True
+event_name = 'hackjohn'
+ifttt_key = 'cJLFuy0CHPBwK0wqndI2g9'
+
+# Create you own IFTTT (https://ifttt.com/create)
+# Select webhooks for this and select receive a web trigger
```

Comment:
```suggestion
# Select webhooks for this and select "Receive a web request".
```

for referencem, here are some URLs that the process took me to:

https://ifttt.com/create/connect-maker_webhooks?sid=3
https://ifttt.com/create/if-receive-a-web-request?sid=9
https://ifttt.com/create/if-receive-a-web-request-then-send-a-rich-notification-from-the-ifttt-app?sid=14